Gopher-Industries/Team-TaskforceX
# 104 getting started

Summary of Changes:

-Created new splash screen images and backgrounds.
-Changed fonts and weights.
-Changed margins to align with wireframe.
-Added additional comments.
-Cleaned up some code
-Changed colours to match wireframe.
-Added and confirmed continue button functions across each getting started screen.
-Added and confirmed back button functions across each getting started screen. (Still need to update type of arrow)
-Added and confirmed continue button pushes user to "accessibility" at the conclusion of getting started.

Testing instructions:

-Confirm background images are visible across all three getting started screens.
-Confirm continue button functions across all three getting started screens.
-Confirm continue button pushes user to "accessibility" at the conclusion of getting started.
-Confirm back button functions across each getting started screen.

![Splash 1](https://user-images.githubusercontent.com/118961659/206852084-1ad005c6-ada9-43c3-a367-f09eacd0ab4a.PNG)
![Splash 2](https://user-images.githubusercontent.com/118961659/206852086-69ef6249-2b3c-4f11-a7ab-f85443850eba.PNG)
![Splash 3](https://user-images.githubusercontent.com/118961659/206852089-ce261d58-afac-43e1-9864-b3ef52e66c46.PNG)
